code/script.js

```diff
@@ -1,7 +1,30 @@
+////// TO-DO //////
+
+//Minimum:
+//[X] re-style CSS! required!
+//[X] Make guess person functionality
+//[X] Make guess person win/lose screen happen
+//[X] fix the "true hat" bug xD
+//[X] consider whether an exception to accessories is the only exception needed for the .category wording issue. Smoking is current also labeled as other, you know :P
+//    [X] reconsider creating a new property for currentQuestion, but refresh yourself on why making an if statement or just NOT importing the category at all might be easier :P consider an alternative source for the text than .category which may already exist :))
+//[X] in filterCharacters, move out the filter method calls from each category message ifs and elses and put them in a single separate if else at the end
+//[_] properly implement the default choice of "choose here" (I think the html is what is left) also credit Maria and the other girl (Maria pinged us?)
+//[_] either keep labels or remove them and find a way to make the category names prettier instead
+ 
+//Extra flare:
+//[X] Add portrait of the secret person on the winOrLose screen
+//[_] Make my friends and memes as characters in a theme overhaul (4 h)
+
+
```

Comment:
I love that you did this to-do in your js. I will do the same next time so I know what I have and haven't done. Great idea 🙌🏻

Reply:
Yes, it is nice, but it should not be found in the final stage of the project.